sdk/azidentity/interactive_browser_credential.go

```diff
@@ -26,6 +26,8 @@ type InteractiveBrowserCredentialOptions struct {
 	ClientSecret string
 	// The redirect URL used to request the authorization code. Must be the same URL that is configured for the App Registration.
 	RedirectURL string
+	// The localhost port for the local server that will be used to redirect back to
```

Comment:
Based on the implementation, you should document that leaving this a zero will default to a random port number.

Reply:
Done.